source/_integrations/devolo_home_network.markdown

```diff
@@ -64,6 +64,9 @@ Currently the following device types within Home Assistant are supported.
 - Number of PLC devices in the same PLC network
   - Updates every 5 minutes
   - Is disabled by default because it typically rarely changes
+- PLC PHY rates
+  - Updates every 5 minutes
+  - PHY rates to/from the device attached to the router are enabled by default, all other disabled
```

Comment:
```suggestion
  - PHY rates to/from the device attached to the router are enabled by default, all other disabled
```
what is meant by "all other"?

Reply:
Let's imagine a PLC network of 4 devices (A, B, C and D). A is the device attached to the router. Then I by default enable PHY rates between A and B, A and C as well as A and D, but disable the PHY rates between B and C, B and D as well as C and D because they are a less interesting connections.

Reply:
OH, I see. Thank you for explaining! So, maybe:
PHY rates to/from the device attached to the router are enabled by default. PHY rates between all other devices are disabled by default.

Reply:
On another note: Not sure if this can fit into the scope of this PR, but _wifi_ should be replaced by _Wi-Fi_.